docs/src/main/sphinx/connector/bigquery.md

```diff
@@ -82,14 +82,6 @@ one for the sales and one for analytics, you can create two properties files in
 having `connector.name=bigquery` but with different `project-id`. This will
 create the two catalogs, `sales` and `analytics` respectively.
 
-### Configuring partitioning
-
-By default the connector creates one partition per 400MB in the table being
-read (before filtering). This should roughly correspond to the maximum number
-of readers supported by the BigQuery Storage API. This can be configured
-explicitly with the `bigquery.parallelism` property. BigQuery may limit the
```

Comment:
I don't see how this was true. if the parallelism was not configured, it defaulted to number of workers